ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/fabric/bgp_base/route-maps.j2

```diff
@@ -54,3 +54,32 @@
 {%         endif %}
 {%     endif %}
 {% endif %}
+{# Route-maps for tenant bgp peers set_ipv4_next_hop parameter #}
+{% if type == "l3leaf" %}
+{%     for tenant in tenants | arista.avd.natural_sort if ( tenant in leaf.filter_tenants or "all" in leaf.filter_tenants ) and leaf.evpn_services_l2_only == false %}
+{%         if tenants[tenant].vrfs is defined %}
+{%             for vrf in tenants[tenant].vrfs | arista.avd.natural_sort if leaf.vrfs is not none and vrf in leaf.vrfs %}
+{%                 if tenants[tenant].vrfs[vrf].bgp_peers is arista.avd.defined %}
+{%                     for bgp_peer in tenants[tenant].vrfs[vrf].bgp_peers %}
+{%                         if tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].nodes is arista.avd.defined %}
+{%                             if inventory_hostname in tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].nodes %}
+{%                                 if tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].set_ipv4_next_hop is arista.avd.defined or tenants[tenant].vrfs[vrf].bgp_peers[bgp_peer].set_ipv6_next_hop is arista.avd.defined %}
+  RM-{{ bgp_peer }}-SET-NEXT-HOP-OUT:
```

Comment:
Please add `{{ vrf }}` in RM name to avoid collision if 2 tenants use same address plan

Reply:
Good idea! Fixed in commit 9e9df78.